src/explore-education-statistics-frontend/src/modules/table-tool/components/TableHeadersForm.tsx

```diff
@@ -24,6 +32,34 @@ export interface TableHeadersFormValues {
   rowGroups: SortableOption[][];
   rows: SortableOption[];
 }
+const removeSiblinglessTotalRows = (
+  categoryFilters: Dictionary<CategoryFilter[]>,
+): CategoryFilter[][] => {
+  return Object.values(categoryFilters).filter(filter => {
+    return filter.length > 1 || !filter[0].isTotal;
+  });
+};
+
+export const returnDefaultTableHeaderConfig = (
```

Comment:
Given that these two functions aren't even being used in `TableHeadersForm`, this should be hoisted up into `TableToolPage` or extracted as some other utility file.